File: Baekjoon/C/2920.c
#include <stdio.h>
int main(){
	int asc=0;
	int des=0;
	
	int arr[9];
	for(int i=0; i<8; i++){
		scanf("%d", &arr[i]);
	}
	
	for(int i=0; i<9; i++){
		if(i+1 == arr[i])
			asc++;
	}
	
	for(int i=8; i>0; i--){
		if(i == arr[8-i])
			des++;
	}
	
	if(asc==8)
		printf("ascending");
	else if(des==8)
		printf("descending");
	else
		printf("mixed");
}

File: Baekjoon/Python/2920.py
n = list(map(int, input().split()))
a = [i for i in range(1, 9)]
d = [i for i in range(8, 0, -1)]

if n == a:
    print('ascending')
elif n == d:
    print('descending')
else:
    print('mixed')
